src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -82,21 +76,16 @@ export const ChatSidebar: React.FC = () => {
         </div>
         <div className="chat-sidebar__tab-content">
           {chatSettings.openedChatType === ChatTypes.VENUE_CHAT && (
-            <VenueChat onAvatarClick={setSelectedProfile} />
+            <VenueChat onAvatarClick={setUserProfile} />
```

Comment:
I don't think you need to prop-drill it. You can just use the hook inside the components

Reply:
👍🏻